core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1930,6 +1930,18 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  def tryClose(value: Any): Unit = {
```

Comment:
This should accept at best `AnyRef`. It doesn't really seem like we need a new global utility method for this. It's a little unusual to try closing things that aren't `Closeable` and we can try to rationalize that in the callers above if possible.